Keyboards/KeyboardsBase/KeyboardViewController.swift

```diff
@@ -1620,7 +1633,8 @@ class KeyboardViewController: UIInputViewController {
       }
     } else {
       letterKeyWidth = (UIScreen.main.bounds.width - 6) / CGFloat(letterKeys[0].count) * 0.9
-      numSymKeyWidth = (UIScreen.main.bounds.width - 6) / CGFloat(numberKeys[0].count) * 0.9
+      //If we are using expanded keys the numberKeys array is empty, we use symbolKeys
```

Comment:
Minor note on comments, @henrikth93: when it's a full line we try to do complete sentences where we'll have a period at the end :) No stress whatsoever though!